### DISCUSSION TITLE
Pedálszög és sebesség/gyorsulás

### DISCUSSION THREAD

**Discussion by KMate-ISD** [SzFMV2022-Tavasz/AutomatedCar-A#79] [Q&A] [answered]
Hello @SzFMV2022-Tavasz/team-a1 ! Pedálszög és sebesség, gyorsulás között milyen összefüggést implementáltatok?

**Comment by molnzsu** [comment, marked as answer]
Szia! A PowerTrain osztály DriveGear metódusábn találod az összefüggéseket. Ott található, hogy milyen gáz és fékpedál állásnál, milyen időközönként és mennyivel változik a sebesség.
